documentation/SRS.md

```diff
@@ -1,3 +1,185 @@
-#Manager for department and employees
+# Manager for department and employees
 
-*coming soon*
\ No newline at end of file
+## Vision
+
+This application is web-application that helps 
+navigate in information about employees and 
+departments. 
+
+Application should provide:
+* Storing information about employees and
+departments in database;
+* Display list of employees in department;
+* Updating employees data (add,editing,delete);
+* Display average cost of employees in a department;
+* Display list of departments;
+* Updating departments data (add,editing,delete);
+* Filtering by date of birthday and period of dates.
+
+## 1. Employees
+### 1.1 Display list about employees
+The mode is designed to view list of employees in department.
+When user selects a department, a list of employees for that 
+department appears **(pic 1.1)**.
+
+#### *Main scenario*
+* User select item Employees;
+* Application open employees list.
+
+![list_employees](./search_period.png)
```

Comment:
In accordance with the requirements, the average salary should be displayed for each department on the Departments page. I like the mockup above, so please don't change anything here. The proposed changes only need to be applied to the Departments page.

Reply:
Fixed